ffi/capi/src/time.rs

```diff
@@ -6,18 +6,22 @@
 pub mod ffi {
     use alloc::boxed::Box;
 
-    use icu_calendar::types::Time;
+    use icu_calendar::Time;
 
     use crate::errors::ffi::ICU4XError;
 
     #[diplomat::opaque]
     /// An ICU4X Time object representing a time in terms of hour, minute, second, nanosecond
-    #[diplomat::rust_link(icu::calendar::types::Time, Struct)]
+    #[diplomat::rust_link(icu::calendar::Time, Struct)]
+    #[diplomat::rust_link(icu::calendar::types::Time, Struct, hidden)]
```

Comment:
Question: is this how we decided to deal with re-exports? Seems cumbersome. Why did you need to duplicate the rust_links in some cases but not others?

Reply:
We should make the tool smart about reexports, really.

Reply:
I lean toward keeping reexports in allowlist.rs instead of spreading the problem out through the bridge code.

Reply:
I agree.